python/feathub/processors/flink/flink_processor.py

```diff
@@ -102,7 +102,14 @@ def __init__(
         except ValueError:
             raise FeathubException("Unsupported deployment mode.")
 
-        if self.deployment_mode == DeploymentMode.SESSION:
+        if self.deployment_mode == DeploymentMode.CLI:
+            self.flink_table_builder = FlinkTableBuilder(
+                self._get_table_env(), self.registry
```

Comment:
Does it mean that the behavior of CLI mode is same as the behavior of Session mode except that `rest.address` and `rest.port` are specified by the user's environment?

If so, I am wondering if it will be simpler to just use the `session` mode instead of adding the `cli` mode. We can still add REAME for users to deploy a FeatHub program configured with `flink session mode` via the Flink CLI tool.